### PR TITLE
[wix-style-react] - stylelint for unsupported browsers

### PR DESCRIPTION
**Pull request by egidijusja**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6642

File: packages/wix-style-react/src/Modal/Modal.st.css [hunk 1]
```diff
@@ -53,7 +53,7 @@
 }
 
 .childrenContainer {
-  -st-states: screen(enum(desktop, mobile, full))
+  -st-states: screen(enum(desktop, mobile, full));
```
Comment (mykas) [inline packages/wix-style-react/src/Modal/Modal.st.css:56]: redudant change

File: packages/wix-style-react/.stylelintrc.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "plugin/no-unsupported-browser-features": [true, {
+      "browsers": [
+        "Chrome >= 76",
+        "Safari >= 12",
+        "Edge >= 18",
+        "Firefox >= 72"
+      ]
+    }]
+  }
+}
```
Comment (mykas) [inline packages/wix-style-react/.stylelintrc.json:15]: Lets add those new lines here

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "typescript",
+    "typescriptreact",
+    "vue",
+    "vue-html",
+    "vue-postcss",
+    "xml",
+    "xsl",
+    "stylable"
+  ]
+}
```
Comment (mykas) [inline .vscode/settings.json:26]: Lets add those new lines here

File: .vscode/extensions.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "stylelint.vscode-stylelint",
+    "esbenp.prettier-vscode",
+    "wix.stylable-intelligence"
+  ]
+}
```
Comment (mykas) [inline .vscode/extensions.json:8]: Lets add those new lines here

File: packages/wix-style-react/package.json [hunk 5]
```diff
@@ -60,6 +60,7 @@
     "postbuild": "npm run copy-public-styles",
     "update-components": ".wuf/update-components.sh",
     "lint": "eslint \"{src,test,testkit,stories}/**/*.js\" --cache --cache-location target/.eslintcache --fix",
+    "stylelint": "stylelint \"./src/**/*.css\"",
```
Comment (mykas) [inline packages/wix-style-react/package.json:63]: Lets not add this script here. Just write it inside the line 182

**Review by mykas — APPROVED** [review]

**Pull request merged**